### DISCUSSION TITLE
Write a simple DNS example using pseudo-spectral solver

### DISCUSSION THREAD

**Discussion by jiweiqi** [DENG-MIT/Arrhenius.jl#8] [Ideas]
Example of non-reacting flow at https://github.com/loiseaujc/NS2D.jl

**Comment by RSuryaNarayan** [comment]
Are we attempting a simple reacting DNS? Then I think a premixed laminar methane-bunsen flame would be a really good idea. We got GRImech30 for the chemistry and we would just need to whip up the CFD aspects of it
